src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Performance/DoNotUseCountWhenAnyCanBeUsedTests.Tests.cs

```diff
@@ -990,6 +992,7 @@ End Function
         Async Sub M()
             Dim b As Boolean
 
+            b = {{|{this.Verifier.DiagnosticId}:(Await GetData().CountAsync).Equals(IntegerZero)|}}
```

Comment:
Same as my previous comment, could you please add this case on the other expressions?